lib/ansible/galaxy/api.py

```diff
@@ -309,10 +325,12 @@ def available_api_versions(self):
         # Calling g_connect will populate self._available_api_versions
         return self._available_api_versions
 
+    @retry_wrapper(jittered_backoff_generator(retries=6, delay=2, max_delay=40), rate_limit_exception)
```

Comment:
It feels like it's best to use explicit kwargs — when I read `rate_limit_exception` I automatically think that it's an instance of some exception object while in fact reading its definition it turns out to be a callable.
And even for the first argument, it'd be useful to know what you set by passing what I assume is an iterator.